src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/player/ModuleAutoWalk.kt

```diff
@@ -31,9 +39,43 @@ import net.ccbluex.liquidbounce.utils.kotlin.EventPriorityConvention.FIRST_PRIOR
  */
 object ModuleAutoWalk : ClientModule("AutoWalk", Category.PLAYER) {
 
-    @Suppress("unused")
-    private val moveInputHandler = handler<MovementInputEvent>(priority = FIRST_PRIORITY) { event ->
-        event.directionalInput = event.directionalInput.copy(forwards = true)
+    private val modes = choices(this, "Mode", Smart, arrayOf(Smart, Simple))
```

Comment:
private val modes = choices(this, "Mode", Smart, arrayOf(Smart, Simple)).apply { tagBy(this) }
optional but this adds a tag of the mode you are using to the arraylist

Reply:
added this (unfunny joke start) super major update (unfunny joke end)

Reply:
i know :skull: